__tests__/commands/install/integration.js

```diff
@@ -452,15 +455,39 @@ test.concurrent(
   },
 );
 
-// disabled to resolve https://github.com/yarnpkg/yarn/pull/1210
-test.skip('install should hoist nested bin scripts', (): Promise<void> => {
-  return runInstall({binLinks: true}, 'install-nested-bin', async (config) => {
+test.concurrent('install should hoist nested bin scripts', (): Promise<void> => {
+  // we mock linkBin to enable us to test it
+  let mockLinkBin;
+
+  const run = buildRun.bind(
+    null,
+    ConsoleReporter,
+    fixturesLoc,
+    async (args, flags, config, reporter, lockfile): Promise<Install> => {
+      const install = new Install(flags, config, reporter, lockfile);
+      const linkBin = install.linker.linkBin;
+      install.linker.linkBin = mockLinkBin = jest.fn((...args) => {
+        return linkBin(...args);
+      });
+      await install.init();
+      await check(config, reporter, {}, []);
+      return install;
```

Comment:
this test needs to verify that bin dependencies were flattened properly.
Especially for a case of conflicting bins